wordpress.org/public_html/wp-content/themes/pub/wporg-support/sass/blocks/_group.scss

```diff
@@ -0,0 +1,14 @@
+.wp-block-group {
+
+	&.is-layout-grid.is-style-cards-grid {
+		grid-template-columns: repeat(auto-fill, minmax(min(32.3%, 100%), 1fr));
+		gap: var(--wp--preset--spacing--10);
+	}
+
+	.wp-block-wporg-link-wrapper {
+
+		#bbpress-forums & {
+			border: 1px solid var(--wp--preset--color--light-grey-1);
```

Comment:
There are a few nasty specificity overrides required

Reply:
Override existing theme styles, bbpress styles or global legacy styles from `wp4.css`? At any rate, a non-issue for me.

Reply:
In this case bbpress, but there are wp4 things to work around too

Reply:
I tried dequeueing wp4 (`wp_dequeue_style( 'wp4-styles' );`), it doesn't seem to change much (though it does fix the local nav alignment), so that's also an option. Probably can't dequeue bbpress, I'm sure we need some of that :)

| With `wp4.css` | Without `wp4.css` |
|---|---|
| ![home-wp4](https://github.com/WordPress/wordpress.org/assets/541093/86e14726-9b33-4f09-80d6-4db1641f9815) | ![home-no-wp4](https://github.com/WordPress/wordpress.org/assets/541093/245d399b-fb13-4fac-9726-c36da3960952) |
| ![forum-wp4](https://github.com/WordPress/wordpress.org/assets/541093/9983c573-023d-4d6d-9fd6-f6dbe85ff0cf) | ![forum-no-wp4](https://github.com/WordPress/wordpress.org/assets/541093/8bd64e16-723d-4e80-90ac-e38d41be8e7c) |